src/lib/homestore_backend/gc_manager.cpp

```diff
@@ -363,7 +370,7 @@ void GCManager::pdev_gc_actor::handle_recovered_gc_task(
                        move_to_chunk);
     }
 
-    if (!process_after_gc_metablk_persisted(gc_task_sb, valid_blob_indexes)) {
+    if (!process_after_gc_metablk_persisted(gc_task_sb, valid_blob_indexes, 0)) {
```

Comment:
nit： do not use magic number 0

Reply:
```
inline static atomic_uint64_t _gc_task_id{1}; // 0 is used for crash recoveryf
```

for the simplicity , 0 is dedicated for crash recovery